recipes/install.rb

```diff
@@ -316,6 +324,26 @@
               }
             }
           },
+          'featureStorePool' => {
```

Comment:
This is ok. *However* during an update, this part of the code won't be executed. So you need to create the connection pool also in `recipes/default`